### PR TITLE
Accessibility requirements on the analysis screen

### PR DESCRIPTION
**Pull request by vizaknaidavid**
feat(GiniCaptureSDK) Add accesibility requirments for image, title and description

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsView.swift [hunk 1]
```diff
@@ -93,15 +93,16 @@ final class CaptureSuggestionsView: UIView {
         bottomConstraint = self.bottomAnchor.constraint(equalTo: superViewBottomAnchor, constant: containerHeight)
         Constraints.active(item: self, attr: .leading, relatedBy: .equal, to: superview, attr: .leading)
         Constraints.active(item: self, attr: .trailing, relatedBy: .equal, to: superview, attr: .trailing)
-        Constraints.active(item: self, attr: .height, relatedBy: .lessThanOrEqual,
+        Constraints.active(item: self, attr: .height, relatedBy: .greaterThanOrEqual,
                            to: nil, attr: .notAnAttribute, constant: containerHeight, priority: 250)
         Constraints.active(constraint: bottomConstraint)
 
+
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsView.swift:100]:
```suggestion
- 
```

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift [hunk 2]
```diff
@@ -22,6 +27,15 @@ final class CaptureSuggestionsViewContainer: UIView {
 
     override func awakeFromNib() {
         super.awakeFromNib()
+
+        isAccessibilityElement = false
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift:31]: @vizaknaidavid would you mind accessibility-related code in a separate method to `configure Accessibility()` ?

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Great work and a few changes are left 😊

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift [hunk 3]
```diff
@@ -11,6 +11,11 @@ final class CaptureSuggestionsViewContainer: UIView {
     @IBOutlet var imageView: UIImageView!
     @IBOutlet var titleLabel: UILabel!
     @IBOutlet var descriptionLabel: UILabel!
+    private var accessibilityImageDescription: String? {
```
Comment (chriskryn) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift:14]: Not sure If there is a need of having this variable ?

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift [hunk 4]
```diff
@@ -31,18 +45,19 @@ final class CaptureSuggestionsViewContainer: UIView {
...
 
-        descriptionLabel.adjustsFontForContentSizeCategory = true
         descriptionLabel.font = configuration.textStyleFonts[.subheadline]
         descriptionLabel.textColor = UIColor.GiniCapture.systemGray
     }
 
     func configureContent(with image: UIImage?, title: String, description: String) {
         self.imageView.image = image
         self.titleLabel.text = title
+        self.titleLabel.accessibilityValue = title
```
Comment (chriskryn) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift:58]: I think there is no need to have accessibilityValue for labels. Then the text is read twice

**Review by chriskryn — CHANGES_REQUESTED** [review]
Screenshot 2022-09-01 at 17 02 50 There seems to be some hidden label, Could you tell me what's the purpose of it?

CaptureSuggestionsView contains some linters waringins could you fix them?
Also, fielprivate keyword seems to be not necessary within CaptureSuggestionsViewContainer

**Review by chriskryn — CHANGES_REQUESTED** [review]
@vizaknaidavid Could you center the loader and the text like in the designs?
Screenshot 2022-09-01 at 17 50 35

**Review by chriskryn — CHANGES_REQUESTED** [review]
I spotted that hideAnimation and showAnimation in AnalysisViewController are not used anywhere. Should we remove it? Beside that it looks good to me :+1:

**Comment by vizaknaidavid** [comment]
> I spotted that hideAnimation and showAnimation in AnalysisViewController are not used anywhere. Should we remove it? Beside that it looks good to me 👍

It is a public function, and I think it could be called from the outside, si that's why I didn't touch it.

**Review by chriskryn — APPROVED** [review]
:+1:

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift [hunk 5]
```diff
@@ -23,26 +23,37 @@ final class CaptureSuggestionsViewContainer: UIView {
     override func awakeFromNib() {
         super.awakeFromNib()
         configureView()
+        configureAccessability()
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsViewContainer.swift:26]: A small typo Accessability-> Accessibility🙈

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Great job, only a small typo to fix 😊

**Review by zladzeyka — APPROVED** [review]
👏🏻🎉👍🏻

**Pull request merged**